scripts/dev/install-pnpm.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
Copied from the coda repo with one change: for some reason I had to do `sudo node` instead of just `node`. No idea why (without the sudo there are permissions errors writing `pnpm` to /usr/local/bin).

Reply:
`make bs` in coda repo runs with sudo already.  Perhaps it doesn't in packs-sdk repo?

Reply:
Ah, that must be it